### ISSUE TITLE
Add Visual Question Answering (VQA) pipeline

### ISSUE THREAD

**Issue by NielsRogge** [huggingface/transformers#17208]
### Feature request

We currently have [ViLT](https://huggingface.co/docs/transformers/model_doc/vilt) in the library, which, among other tasks, is capable of performing visual question answering (VQA).

It would be great to have a pipeline for this task, with the following API:

```
from transformers import pipeline

pipe = pipeline("vqa")
pipe("cats.png", "how many cats are there?")
```

This pipeline could default to the https://huggingface.co/dandelin/vilt-b32-finetuned-vqa checkpoint. 

This can be implemented similar to [other pipelines](https://github.com/huggingface/transformers/tree/main/src/transformers/pipelines).

### Motivation

A pipeline is required in order to have inference widgets + a task defined at hf.co/tasks.

Moreover, it would be great to do VQA in two lines of code.

### Your contribution

I can definitely assist in this, together with @Narsil, who's the pipeline expert.

**Comment by Narsil** [comment]
Tagging @mishig25 for the widget

**Comment by LysandreJik** [comment]
Also LXMERT should handle this task, but likely has a very different API.

**Comment by mishig25** [comment]
This sounds amazing. Happy to contribute in anyway I can

**Comment by sijunhe** [comment]
I'd love to pick this up!

**Comment by sabarish-srinivasan** [comment]
Hey @sijunhe, I'm just starting out in open-source, but I'd like to help out however I can!

**Comment by sijunhe** [comment]
@sabarish-srinivasan appreciate the help but I saw this a little late and I am almost done with the PR.

**Comment by sabarish-srinivasan** [comment]
@sijunhe No problem, thanks for letting me know!

**Comment by sijunhe** [comment]
@LysandreJik I looked at both ViLT and LXMERT and I don't think it's possible to combine these two into a single pipeline for the following reasons:

1. ViLT formats VQA as a classification task and LXMERT formats VQA as a squad-like QA task. It'd be hard to write a common post-processing
2. ViLT is self-contained within transformers but LXMERT expects some faster-RCNN model to generate the visual features that goes into the model.

**Comment by NielsRogge** [comment]
Yes, don't think we should support LXMERT for the pipeline, since it isn't entirely included in the Transformers library.

**Comment by sijunhe** [comment]
Now that #17286 is merged, this issue should be closed now?
